submitter/relayer/relayer.go

```diff
@@ -23,10 +24,11 @@ import (
 
 type Relayer struct {
 	btcclient.BTCWallet
-	sentCheckpoints  types.SentCheckpoints
-	tag              btctxformatter.BabylonTag
-	version          btctxformatter.FormatVersion
-	submitterAddress sdk.AccAddress
+	submittedCheckpoints map[uint64]*types.CheckpointInfo
+	tag                  btctxformatter.BabylonTag
+	version              btctxformatter.FormatVersion
+	submitterAddress     sdk.AccAddress
+	resendIntervals      uint
```

Comment:
```suggestion
	resendIntervalSeconds      uint
```

Reply:
Another way (that might be a bit cleaner) is
```suggestion
	resendIntervals      time.Duration
```

Reply:
I agree that changing the interval to `time.Duration` is cleaner and more flexible, but it increases the chance of having invalid configurations as it requires the time unit specified, for example, `100s` vs. `100`.

Reply:
I don't think that's a big issue, we already do that in other places as well (e.g. the API).